getting-started/cloud-multi-node.md

```diff
@@ -0,0 +1,250 @@
+>:TIP:Timescale currently offers two options for cloud hosting. If you are a
+Timescale Forge user, please use the documentation for [setting up multi-node
+on Forge][forge-multi-node] instead.
+
+# Setting up TimescaleDB 2.0 Multi-node on Timescale Cloud
+
+TimescaleDB 2.0  [introduces a number of new features][changes-in-tsdb2] 
+to supercharge time-series data even further. One of the most anticipated new features 
+is what we call **multi-node** - the ability to create a cluster of TimescaleDB 
+instances to scale both reads and writes. 
+
+In this how-to, we’ll show you how to create a multi-node cluster in your Timescale 
+Cloud account with TimescaleDB 2.0 as a "do-it-yourself" (DIY) multi-node experience.
```

Comment:
"do-it-yourself" (DIY) multi-node experience
 
Let's check with Prashant or maybe mike about this phrasing. I belive there is some sensitivity around how "GA" or how "supportable" multinode is. So I would want to stick to whatever our policy is there

Reply:
For now, this was what @mfreed settled on so I think we'll keep and consider when either environment supports something smoother.